profiles/common/modules/features/ecas_env/ecas_env.module

```diff
@@ -20,7 +20,7 @@ function ecas_env_menu_link_alter(&$item) {
     if (!isset($item['customized']) || (isset($item['customized']) && $item['customized'] == 0)) {
       switch ($item['link_path']) {
         case 'ecas':
-          $item['options']['attributes']['title'] = t('Log in');
+          $item['options']['attributes']['title'] = t('EU login');
```

Comment:
This change comes to remark about the fact that Log in is confusing for site where EU Login is not used for contributors.